### ISSUE TITLE
MueLu: expose all verbosity options in "easy" interface

### ISSUE THREAD

**Issue by jhux2** [trilinos/Trilinos#2149]
The only verbosity levels available through MueLu's "easy" parameter list interpreter, the one used by Nalu, are "none", "low", "medium", "high", "extreme", and "test".

The std::map of verbosity strings and corresponding enumerated types should be defined in one place, then all code can use this map.

**Comment by aprokop** [comment]
Sorry, I don't quite understand what you are saying. I thought Nalu passed MueLu options directly to MueLu? This way MueLu's interpreter is the only place to understand those options.

**Comment by jhux2** [comment]
@aprokop You're right.  I'm referring to MueLu's internals.  The interpreter itself only understands six of the many verbosity options.  It would be really nice, for instance, to be able to pass in "Timings" to enable MueLu's timers but have no setup details print.

**Comment by jhux2** [comment]
Btw, this isn't hard, I opened this so I don't forget.

**Comment by aprokop** [comment]
Oh, I see. So you want finer granularity, and to be able to pass something like "low | timings".

**Comment by jhux2** [comment]
Actually, not quite that level of granularity.  I want to be able to pass these through, which come from MueLu_ParameterListInterpreter_def.hpp:
```
      std::map<std::string, MsgType> verbMap;
      //for developers
      verbMap["Errors"]         = Errors;
      verbMap["Warnings0"]      = Warnings0;
      verbMap["Warnings00"]     = Warnings00;
      verbMap["Warnings1"]      = Warnings1;
      verbMap["PerfWarnings"]   = PerfWarnings;
      verbMap["Runtime0"]       = Runtime0;
      verbMap["Runtime1"]       = Runtime1;
      verbMap["RuntimeTimings"] = RuntimeTimings;
      verbMap["NoTimeReport"]   = NoTimeReport;
      verbMap["Parameters0"]    = Parameters0;
      verbMap["Parameters1"]    = Parameters1;
      verbMap["Statistics0"]    = Statistics0;
      verbMap["Statistics1"]    = Statistics1;
      verbMap["Timings0"]       = Timings0;
      verbMap["Timings1"]       = Timings1;
      verbMap["TimingsByLevel"] = TimingsByLevel;
      verbMap["External"]       = External;
      verbMap["Debug"]          = Debug;
      verbMap["Test"]           = Test;
      //for users and developers
      verbMap["None"]           = None;
      verbMap["Low"]            = Low;
      verbMap["Medium"]         = Medium;
      verbMap["High"]           = High;
      verbMap["Extreme"]        = Extreme;
```

**Comment by aprokop** [comment]
How would you combine them, though? Each one by itself is pretty useless, you would want to combine a few from different areas, which is what the "low" and friends do. I think that maybe the easier way is to just add the timings, as I can see situations where you just want the timers output without anything else.

**Comment by cgcgcg** [comment]
Fixed via #7336
